### DISCUSSION TITLE
Welcome to AI-Playground Discussions!

### DISCUSSION THREAD

**Discussion by Nuullll** [intel/AI-Playground#34] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bobduffy** [comment]
Thanks for adding. Adding a link here to our Discord channels on AI Playground

**Reply by brownplayer** [reply]
An error occurred while loading mistral-nemo. What is the cause

**Reply by ccc0ra** [reply]
> Thanks for adding. Adding a link here to our Discord channels on AI Playground

Hello,
A growing number of Intel Arc users are facing **severe instability and a noticeable drop in image quality** since the July updates: identical seed, identical parameters — yet completely different outputs after a every reboot, and the overall quality is **significantly worse than before July**.. This is breaking one of the core foundations of SDXL workflows.

The evidence points to **dynamic kernel recompilation** in oneAPI/WGPU and **forced dtype changes** introduced by the AIPG v3 wrapper. These decisions directly undermine SDXL + LoRA reproducibility and make serious work impossible with very precise loras/prompts associations.

We urgently need:
- **backend override controls** (disable‑wgpu / disable‑oneapi),
- a **truly stable build** that prioritizes FP32/BF16 for all critical inference paths,
- **clear rollback instructions** for users stuck with irreversible driver changes (even restoration couldn't help.

Without immediate action, you risk a **major loss of trust** from the community and **a continued decline in SDXL quality** on Intel hardware.
Thank you.

**Comment by majerus1223** [comment]
hey

**Comment by lh3953** [comment]
在打开程序的时候 一直提示AI playground loading，怎么解决？

**Reply by brownplayer** [reply]
按照官方的说法就是
![image](https://github.com/user-attachments/assets/27edb251-b51d-47a7-ba7c-d03ff12a3bfd)
这个卸了重新装

**Reply by Nuullll** [reply]
请根据[这里](https://github.com/intel/AI-Playground/issues/57#issuecomment-2398761243)的步骤查看完整错误信息，大概率是因为网络问题环境没安装好。

**Comment by yeetecer** [comment]
Backend initialization failed
The AI inference backend failed to initialize. Please try restarting the application. If the problem persists, you can check the Details for additional information about the error.

**Comment by lh3953** [comment]
我已经尝试重新安装这个软件，也重新安装过系统，还重新安装了显卡驱动。但是还是没办法解决，这个问题的根源是软件安装失败？还是我电脑网络问题？或者是我的系统（win10）不能使用这个app？

**Reply by shira-zusa** [reply]
which serials GPU  do you use? I can run it with B580 and amd 5600

**Comment by FAIOJS** [comment]
组件安装特别慢，有什么方法可以加快吗？

**Comment by Xtiger909** [comment]
如何修改模型的默认目录？ 为何自行指定模型目录后就提示错误，路径只有英文。
How to modify the default directory of the model? Why does specifying a custom model directory result in an error, and why does the path only accept English characters?

**Comment by WeihanLi** [comment]
Would be great to support Ollama compatible API and the winget package install/upgrade

**Reply by WeihanLi** [reply]
seeing that there's a portable `ollama`, not sure if it could be integrated with the AI-Playground

https://github.com/intel/ipex-llm/blob/main/docs/mddocs/Quickstart/ollama_portable_zip_quickstart.md

**Comment by zoner72** [comment]
it is so fast! even on my A380
Is there a server API, eg rest interface localhost:port?

**Comment by shira-zusa** [comment]
I want change my image save path,but i dont how to do?someone can heple me,thanks.

**Comment by lh3953** [comment]
cpu：13600KF, mobo：B660..&nbsp; GPU:A770&nbsp; &nbsp; &nbsp;The GPU is also produced by Intel itself, which is why I want to use this software. However, I always can't make it work.After installing the latest version, I have now resolved the issue of being unable to log in. The new problem is that I can't download the large model. In the software interface, there is a prompt asking me to download the model file, but the size of the model doesn't show up, and I can't click to download it. I thought it was a problem with my network, but even after using a VPN, it still doesn't work.

**Reply by bobduffy** [reply]
I'll alert the dev team. I believe there should be a time out that allows you to confirm the terms under a slow network route

**Comment by lh3953** [comment]
应该是能用，我试一下。主要是这个主程序无法在程序内直接下载模型，点击下载的按钮直接就是灰色的，我用了vpn和加速器都不行。比较烦。。

**Comment by shira-zusa** [comment]
what happening with my GPU? could someone help me fix it
![Uploading 屏幕截图 2025-08-23 143500.png…]()

**Comment by salmonlips** [comment]
I'm loving your software, i just got gpt-oss-20b up running, is ther a way to hide the thinking context or minimize it, and then have the output?  or is that in the plans?
it's a bit difficult parsing where it stops thinking and where it starts replying

**Reply by salmonlips** [reply]
side note on that same subject within the gguf tab i can't load documents to refernece either they just spin out stall, openvino is fine but basically can only get llama 3.1 running of anything newer

**Reply by bobduffy** [reply]
When models reason, you can hide the reasoning.  GPT-OSS sometimes carries reasoning to its answer.

For embedding document first try something simple. Put your name in a .txt file. Attach and see ask it what your name is.  

Then expand from there

**Reply by salmonlips** [reply]
that helped, i find gpt is always carrying reasoninginto the answer and then putting out the answer, it's a good answer much better than llama and phi haha but it's, still hard to parse
you ended up being right about the txt file as well i think i was throwing something too big at it, in the openvino side of the models it could handle it no issue but on this side its' nto happening

**Comment by gnaw0725** [comment]
When I try to install the AI Playground component, it consistently fails with the following error message. My graphics card is an Intel Arc, and both the GPU and NPU drivers have been updated to the latest version.





[device-service] {
    "device_list": [
        {
            "device_id": 0,
            "device_name": "Intel(R) Arc(TM) Graphics",
            "device_type": "GPU",
            "pci_bdf_address": "0000:00:02.0",
            "pci_device_id": "0x7d55",
            "uuid": "00000000-0000-0200-0000-00087d558086",
            "vendor_name": "Intel(R) Corporation"
        }
    ]
}

index-wNbU7XkZ.js:156 [ai-backend] Couldn't detect best device, selecting first. Error: TypeError: Cannot read properties of undefined (reading 'id')
（匿名） @ index-wNbU7XkZ.js:156
（匿名） @ VM5:2
emit @ VM4 sandbox_bundle:2
onMessage @ VM4 sandbox_bundle:2
index-wNbU7XkZ.js:13 Error: Error invoking remote method 'detectDevices': TypeError: Cannot read properties of undefined (reading 'name')
zR @ index-wNbU7XkZ.js:13
Xo @ index-wNbU7XkZ.js:13
（匿名） @ index-wNbU7XkZ.js:13
Promise.catch
cr @ index-wNbU7XkZ.js:13
cr @ index-wNbU7XkZ.js:13
n @ index-wNbU7XkZ.js:17

**Reply by bobduffy** [reply]
Submit your issue to the issues tab. Provide information about your system. Also the true issue might be an error just before this output

**Comment by calvinwonghk** [comment]
does it support intel core ultra 9 285k?

**Comment by d1s1rtfenec-commits** [comment]
it seems only dedicated for Windows OS ? why ? you code it in VBA ? please provide asap a decent tool for any 6.x modern kernel :)

**Comment by shira-zusa** [comment]
I don't know why versions 3.0.0 and 3.0.1 can't launch the ComfyUI backend instantly like version 2.6.2. It's really troublesome. I like this software a lot, but the user experience with version 3.0.0 has been far from satisfactory.

**Reply by bobduffy** [reply]
That's not the experience across dozens of tested devices. ComfyUI should work the same. Post to issue and explain the issue.

**Comment by dunkeng** [comment]
The network connection here is really poor, so I downloaded the model myself and put it into the model directory.But how do I modify the following parameters?
json
{
  "name": "unsloth/Qwen3.5-9B-GGUF/Qwen3.5-9B-Q4_K_M.gguf",
  "mmproj": "unsloth/Qwen3.5-9B-GGUF/mmproj-BF16.gguf",
  "type": "llamaCPP",
  "supportsToolCalling": true,
  "supportsVision": true,
  "supportsReasoning": true,
  "maxContextSize": 262144
},
The built-in models have corresponding configuration files, but I can’t find any config file for the model I downloaded manually, nor can I find an option to edit these settings in the interface.

**Comment by Andrarch** [comment]
Is there a guide/setup to use it as an LLM for other tools like agents? To use this similar to ollama or lmstudio as a server? Or is there a CLI?

**Comment by bobduffy** [comment]
You can use Agentic Chat setting to add MCP servers. Once added use a tool calling model. The model will be aware of thenspp and able to control it. You can search the web for MCP servers to allow you to control many popular apps.

Next week we release a feature to do this from Telegram app on your phone. See our3.1.1b release notes, once released

Installing and configuring autonomous agents is not yet integrated

**Comment by ccc0ra** [comment]
> ## 👋 Welcome!
> 
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
>     * Ask questions you’re wondering about.
> 
>     * Share ideas.
> 
>     * Engage with other community members.
> 
>     * Welcome others and are open-minded. Remember that this is a community we
>       build together 💪.
> 
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Hello,
I would like to report that since the release of AIPG v3 (I am currently on v3.2.0‑beta), the platform has become increasingly unstable on my system.

Each time I start AIPG, and each time I restart ComfyUI from the Setup Wizard panel, I obtain different results when generating the same SDXL image with the same parameters. Even when I switch parameters in the Setup Wizard and then revert to the previous ones, the output changes again. At this point, it is impossible to maintain a stable setup.

I had already experienced a major drop in image quality after updating to AIPG v3 and installing the July Intel drivers and the latest oneAPI components (Lenovo Intel Aura Edition – Intel® Core™ Ultra 7 255H + Intel® Arc™ 140T GPU). But with v3.2.0‑beta, the situation has become even worse.

On top of that, I was wrongfully banned from the AIPG Discord, even though I am one of the few people using AIPG for real creative work rather than quick tests. Criticism does not seem to be tolerated, and it feels like the project is moving backwards, despite previously outperforming several Chinese competitors — at least with SDXL and my pre‑July 2026 configuration, where I consistently obtained better results than on their hardware (even if my laptop was not very fast).

Please take these issues seriously and support the people who are doing good work instead of discouraging them. AIPG had enormous potential, and I hope it can return to the level of quality it once had.

Thank you.
